tasks/ThrusterTask.cpp

```diff
@@ -95,14 +93,13 @@ void ThrusterTask::cleanupHook()
     node->Fini();
 }
 
-void ThrusterTask::setGazeboModel(WorldPtr _world,  ModelPtr _model, sdf::ElementPtr _plugin)
+void ThrusterTask::setGazeboThruster( ModelPtr model )
```

Comment:
Why this change ? I'd like all tasks to have the same interface, it would make it easier to switch to a true plugin-based system later on.